model/task/db.go

```diff
@@ -579,6 +579,17 @@ func ByPreviousCommit(buildVariant, displayName, project, requester string, orde
 	}
 }
 
+// ByPreviousCommit creates a query on Evergreen as the requester on a previous revision with the same buildVariant, displayName and project
```

Comment:
nit: period at the end of doc comment sentence.

Reply:
This doc comment is also kind of the inverse of what it should be - it's implying how it's actually used under the assumption that it's called by `GetTaskIdBetweenTasks`, when it should be stating what it is querying for in a more general way. In this case, this is just an equality query on the given fields, so I would phrase it as something like `returns a query for the given requester, BV, etc.`.